postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -234,6 +243,25 @@ def _collect_statement_samples(self):
             tags=self._tags,
         )
 
+    def _obfuscate_query(self, row):
+        try:
+            return datadog_agent.obfuscate_sql(row['query'], self._obfuscate_options)
+        except Exception as e:
+            self._log.debug("Failed to obfuscate statement: %s", e)
+            self._check.count(
+                "dd.postgres.statement_samples.error", 1, tags=self._dbtags(row['datname'], "error:sql-obfuscate")
+            )
+
+    @staticmethod
+    def _collect_active_query(row, query_signature):
```

Comment:
Nit: this isn't "collecting" anything like `collect_plan`, which makes another query. 

Could call this `to_active_query`?